config/prod/iatlas-api.yaml

```diff
@@ -0,0 +1,34 @@
+template_path: ecs.yaml
+stack_name: iatlas-prod
+dependencies:
+  - common/iatlasvpc.yaml
+  - prod/iatlas-kms.yaml
+  - prod/iatlas-db.yaml
+  - prod/iatlas-api-hostedzone.yaml
+  - prod/iatlas-api-certificate.yaml
+stack_tags: {{stack_group_config.stack_tags}}
+parameters:
+  InstanceType: 't3a.medium'
+  KeyName: 'iatlas-api-prod' # For this to work, this key must be added in the SAGE AWS.
+  VpcId: !stack_output_external iatlasvpc::VPCId
+  BastionHostSgId: !stack_output_external iatlasvpc::BastianSecurityGroup
+  PublicSubnetIds:
+    - !stack_output_external iatlasvpc::PublicSubnet
+    - !stack_output_external iatlasvpc::PublicSubnet1
+    - !stack_output_external iatlasvpc::PublicSubnet2
+  PrivateSubnetIds:
+    - !stack_output_external iatlasvpc::PrivateSubnet
+    - !stack_output_external iatlasvpc::PrivateSubnet1
+    - !stack_output_external iatlasvpc::PrivateSubnet2
+  MaxSize: '2'
```

Comment:
how large do you think this will be in production?  I think the current VPC created for iAtlas is pretty small (0/24) which means only (0/27) 32 instances max per subnet.  if you think we need more than that then we need to update the VPC.  The VPC template is here: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/VPC/vpc-mini.yaml